stories/rating-indicator/rating-indicator.stories.js

```diff
@@ -345,9 +345,42 @@ DisplayMode.storyName = 'Display mode';
 DisplayMode.parameters = {
     docs: {
         storyDescription: `
-Data can be presented as label-value field pairs without editable fields in display-only forms, 
-see <a href="form-grid.html">Form Grid</a> for more details. If you want to include a rating indicator in a display-only form, 
-add the \`.is-display-mode\` to the \`fd-rating-indicator\` element.
+Data can be presented as label-value field pairs without editable fields in display-only forms, see <a href="form-grid.html">Form Grid</a> for more details.
+If you want to include a rating indicator in a display-only form, add the \`.is-display-mode\` to the \`fd-rating-indicator\` element.
```

Comment:
We will need an example with `display-mode` modifier class to make the screenshot and detect future changes/bugs.